pkgs/applications/accessibility/contrast/default.nix

```
@@ -39,6 +44,9 @@ rustPlatform.buildRustPackage rec {
     ninja
     pkg-config
     python3
+    rustPlatform.rust.cargo
+    rustPlatform.cargoSetupHook
+    rustPlatform.rust.rustc
     wrapGAppsHook
     glib # for glib-compile-resources
   ];
```

```suggestion
    wrapGAppsHook
    glib # for glib-compile-resources
  ] ++ (with rustPlatform; [
    rust.cargo
    cargoSetupHook
    rust.rustc
  ];
```

This could be done for all 4 packages.

---

For just three packages it just looks too clunky IMO so I tend to avoid `with` in my packages unless there are much more dependencies.